test/verify/check-metrics

```diff
@@ -368,6 +368,38 @@ class TestHistoryMetrics(MachineCase):
                 });
             })"""))
 
+            b.logout()
+
+        #
+        # Journal logs
+        #
+
+        # Journal was recorded on Fedora 33 and when trying to use it with older systemd it fails with:
+        # `Journal file /var/log/journal/*/journal.journal uses an unsupported feature, ignoring file.`
+        if self.machine.image in ["centos-8-stream", "rhel-8-4", "ubuntu-2004", "ubuntu-stable", "debian-stable", "debian-testing"]:
```

Comment:
Debian testing has systemd 247.3, ubuntu-stable has 246. These ought to be sufficiently new. But this is not that important indeed.